README.md

```diff
@@ -34,6 +34,20 @@ To effectively migrate the addon or port its commits, use the `--fork` option:
 
     $ oca-port 14.0 15.0 shopfloor --fork camptocamp
 
+You can also directly blacklist a bunch of PRs on a given branch thanks to the
+`oca-port-pr` tool:
+
+    $ oca-port-pr blacklist 250,251 15.0 shopfloor
+
+You could give a more detailed reason of this blacklist with `--reason` parameter:
+
+    $ oca-port-pr blacklist 250,251 15.0 shopfloor --reason "Refactored in 15.0, not needed anymore"
+
+And if the module has been moved to another repository, you can specify its remote as well:
+
+    $ git remote add new_repo git@github.com:OCA/new-repo.git
+    $ oca-port-pr blacklist 250,251 15.0 shopfloor --remote new_repo
```

Comment:
We could also write it `oca-port-pr blacklist 250,251 new_repo/15.0 shopfloor`, it's already supported by the `Branch` constructor: https://github.com/OCA/oca-port/blob/main/oca_port/utils/git.py#L20-L21
But maybe this has to be removed while working on https://github.com/OCA/oca-port/pull/33 to have only one syntax everywhere.

Reply:
needs rework after https://github.com/OCA/oca-port/pull/35